docs/en/connector-v2/EnvConf.md

```diff
@@ -0,0 +1,19 @@
+# EnvConf
+
+This document describes env configuration information,env unifies the environment variables of all engines.
+
+## jars
+
+Third-party packages can be loaded via `jars`, like `jars="file://local/jar1.jar;file://local/jar2.jar"`
+
+## job.mode
+
+You can configure whether the task is in batch mode or stream mode through `job.mode`, like `job.mode = "BATCH"` or `job.mode = "STREAMING"` 
+
+## execution.checkpoint.interval
+
+Gets the interval in which checkpoints are periodically scheduled.
+
+## execution.parallelism
```

Comment:
change to `parallelism` because https://github.com/apache/incubator-seatunnel/pull/3353/files#diff-6f68c25dd21780017dc98d220278af32f5ced954e0a0cd4ef3f585fd3f85380e.